scipy/sparse/linalg/_dev_scripts/check_normcond_behavior.py

```diff
@@ -0,0 +1,295 @@
+#!/usr/bin/env python3
```

Comment:
This file needs to be converted to actual tests such that they always run in case they are not exact copies of the tests below. We don't keep additional development versions of the designs.